app/main/forms/suppliers.py

```diff
@@ -130,46 +130,46 @@ def validate(self):
         if not super(AddCompanyRegistrationNumberForm, self).validate():
             valid = False
         if self.has_companies_house_number.data == "Yes" and not self.companies_house_number.data:
-            self.companies_house_number.errors.append('You must enter a Companies House number.')
+            self.companies_house_number.errors.append('Enter a Companies House number.')
             valid = False
 
         if self.has_companies_house_number.data == "No" and not self.other_company_registration_number.data:
-            self.other_company_registration_number.errors.append('You must provide an answer.')
+            self.other_company_registration_number.errors.append('Enter a company registration number.')
             valid = False
 
         return valid
 
 
 class CompanyPublicContactInformationForm(FlaskForm):
     company_name = DMStripWhitespaceStringField('Company name', validators=[
-        InputRequired(message="You must provide a company name."),
-        Length(max=255, message="You must provide a company name under 256 characters.")
+        InputRequired(message="Enter a company name."),
+        Length(max=255, message="Enter a company name under 256 characters.")
     ])
```

Comment:
Change to:

Enter your company number
Enter your company name under 256 characters